rlclientlib/azure_factories.cc

```diff
@@ -15,11 +18,15 @@ namespace reinforcement_learning {
   int observation_sender_create(i_sender** retval, const u::configuration&, error_callback_fn*, i_trace* trace_logger, api_status* status);
   int interaction_sender_create(i_sender** retval, const u::configuration&, error_callback_fn*, i_trace* trace_logger, api_status* status);
   int decision_sender_create(i_sender** retval, const u::configuration&, error_callback_fn*, i_trace* trace_logger, api_status* status);
-
+  int observation_api_sender_create(i_sender** retval, const u::configuration& cfg, error_callback_fn* error_cb, i_trace* trace_logger, api_status* status);
```

Comment:
I think it would be helpful to add a comment explaining the difference between interaction/observation_sender_create and interaction/observation_api_sender_create

Reply:
Added comments to these functions.